src/collective/easyform/tests/testMailer.py

```diff
@@ -543,7 +546,7 @@ def test_MailerXMLAttachments(self):
         output_nodes = (
             b'<field name="replyto">test@test.org</field>',
             b'<field name="topic">test subject</field>',
-            b'<field name="richtext">Raw</field>',
+            b'<field name="richtext">%s</field>' % very_long_line,
             b'<field name="comments">test comments\xf0\x9f\x98\x80</field>',
         )
 
```

Comment:
test the output of get_mail_text to make sure it has base64 encoded the XML attachment